src/languages/scala.js

```diff
@@ -29,23 +29,24 @@ export default function(hljs) {
   const STRING = {
     className: 'string',
     variants: [
+      {
+        begin: '"""',
+        end: '"""'
+      },
       {
         begin: '"',
         end: '"',
         illegal: '\\n',
         contains: [ hljs.BACKSLASH_ESCAPE ]
       },
-      {
-        begin: '"""',
-        end: '"""',
-        relevance: 10
```

Comment:
Basically the main change I see here is that you removed the `relevance`, and that you changed the order. Can you please explain what do those changes do? Next time I might just contribute it myself :)

Reply:
Well if `"` matches first that breaks everything, so we have to match `"""` first... and the relevance is a much longer explication... find the thread on huge relevance and read it for a start for more thoughts... 10 is just too high for something that is shared by several languages now, so I removed the huge relevance bump.

_The sequencing is what fixes the issue._